public/featured.js

```diff
@@ -0,0 +1,90 @@
+// path variables
+let path = window.location.pathname
+let pathArray = path.split('/')
+let id = pathArray.pop()
+console.log(id)
+//  funcitoning variables
+let title = document.getElementById('featured')
+title.textContent = id
+let send = document.getElementById('send')
+// commenting variables
+let input = document.getElementById('input-name')
+let inputValue = input.value
+let email = document.getElementById('input-email')
+let emailValue = email.value
+let comment = document.getElementById('comment-area')
+let commentValue = comment.value
+
+// -----------enter name notes number address hours into sidebar-----
+window.fetch('https://json-server.burlingtoncodeacademy.now.sh/restaurants/' + id)
+    .then(response => response.json()) /* get date */
+    .then(json => {
+        let name = document.getElementById('name')
+        let notes = document.getElementById('notes')
+        let number = document.getElementById('number')
+        let address = document.getElementById('address')
+        let hours = document.getElementById('hours')
+        name.textContent = json.name
+        notes.textContent = json.notes.pop(0)
+        number.textContent = json.phone
+        address.textContent = json.address
+        hours.textContent = json.hours
+        dropPin(address.textContent, name.textContent)
+
+    }
+    
+    )
+   
+    // -------------------drop pin and focus on location -----------------------------
+  function dropPin(address, name){  
+let latLon = {}
+    fetch(`https://nominatim.openstreetmap.org/search/?q=${address}&format=json`)
+.then((data) => {
+    return data.json()
+})
+.then((locInfo) => {
+    let info = locInfo[0]
+    console.log(locInfo[0])
+    let lat = info.lat
+    let lon = info.lon
+    latLon.lat = lat
+    latLon.lon = lon
+    let mymap = L.map('myMap').setView([lat, lon], 18)
+    var OpenStreetMap_HOT = L.tileLayer('https://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png', {
```

Comment:
Because you're using the French map whenever you visit your `featured` page google asks if it should translate the page. Nothing major, but a funny little behavioral quirk I thought I'd mention.